Dockerfile

```diff
@@ -21,10 +21,21 @@ FROM alpine:${ALPINE_VERSION} as base-release
 RUN apk --update --no-cache add ca-certificates && update-ca-certificates
 ENTRYPOINT ["/bin/helmwave"]
 
+### Base image with shell and debugging tools
+FROM alpine:${ALPINE_VERSION} as base-debug-release
+RUN apk --update --no-cache add ca-certificates jq bash &&\
+	update-ca-certificates
```

Comment:
ca-certificates are updated automatically during install/upgrade

Reply:
I saw that in older targets :)